tests/test_lodcm.py

```diff
@@ -1,20 +1,35 @@
 import logging
-from unittest.mock import Mock
+from unittest.mock import Mock, patch
 
 import pytest
+import numpy as np
 from ophyd.sim import make_fake_device
 
-from pcdsdevices.lodcm import H1N, LODCM, Dectris, Diode, Foil, YagLom
+from pcdsdevices.lodcm import (H1N, LODCM, Dectris, Diode, Foil, YagLom, Y1,
+                               CHI1, Y2, CHI2, H2N, SimFirstTower,
+                               SimSecondTower, LODCMEnergySi, LODCMEnergyC)
 
 logger = logging.getLogger(__name__)
 
 
+def motor_setup(mot, pos=0):
+    mot.user_readback.sim_put(0)
+    mot.user_setpoint.sim_put(0)
+    mot.user_setpoint.sim_set_limits((0, 0))
```

Comment:
1. If we don't use a fake motor (like FastMotor) for example, we have to fake a few things in order to be able to test with a motor, these below I had to setup specifically for the IMS motor, I think they don't all require the same things:
```
user_readback
user_setpoint
limits
motor_spg # to 2 - because unable to move otherwise, 2 - Go
```

2. The limits are mostly set here so we make `check_value` from `conftest` happy, otherwise we'll get a type error when trying to get the limits:
```
 low_limit, high_limit = self.limits
```
As far as limits(0, 0) - I am not sure how that works with real motors and the limits for those motors, but when testing - yes, it does seem like (0, 0) means `NO LIMITS` - or unlimited travel like you said - I could be wrong here though